src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -84,4 +100,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     response.sendRedirect("/user-page.html?user=" + recipient);
   }
+
+  // Translates text into target language
+  private void translateMessages(List<Message> messages, String targetLanguageCode) {
+    Translate translate = TranslateOptions.getDefaultInstance().getService();
+
+    for (Message message : messages) {
+      String originalText = message.getText();
+
+      Translation translation =
+          translate.translate(originalText, TranslateOption.targetLanguage(targetLanguageCode));
+      String translatedText = translation.getTranslatedText();
+
+      message.setText(translatedText);
+    }
+  }
+
+  private void setText(String translatedText) {
```

Comment:
Where do you use this method?

Reply:
In the process of implementing .setText at line 115